Cubical/Foundations/CartesianKanOps.agda

```diff
@@ -2,9 +2,12 @@
 {-# OPTIONS --safe #-}
 module Cubical.Foundations.CartesianKanOps where
 
-open import Cubical.Foundations.Prelude
-open import Cubical.Foundations.GroupoidLaws
-open import Cubical.Foundations.Transport
+open import Cubical.Core.Everything
+open import Cubical.Core.Interpolate
```

Comment:
Why import both Everything and the separate file? Wouldn't it make more sense to put interpolate in Prelude and then you don't need to make a local definition of refl?